### PR TITLE
doc(network): remove references to outdated NetworkPolicy examples

### PR DESCRIPTION
**Pull request by lahwaacz**
See https://github.com/longhorn/longhorn/pull/9077 for context.

### CODE REVIEW

File: content/docs/1.5.0/references/networking.md [hunk 1]
```diff
@@ -8,10 +8,10 @@ weight: 3
 This page documents the networking communication between components in the Longhorn system. Using this information, users can write Kubernetes NetworkPolicy
 to control the inbound/outbound traffic to/from Longhorn components. This helps to reduce the damage when a malicious pod breaks into the in-cluster network.
 
-We have provided some NetworkPolicy example yamls at [here](https://github.com/longhorn/longhorn/tree/master/examples/network-policy).
-Or you can enable the setting in the helm chart to install these NetworkPolicy [https://github.com/longhorn/longhorn/blob/master/chart/values.yaml]
+The helm chart will install NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
+The manifests of these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```
Comment (jhkrug) [inline content/docs/1.5.0/references/networking.md:12]:
```suggestion
The helm chart installs NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
Manifests for these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```

File: content/docs/1.5.1/references/networking.md [hunk 2]
```diff
@@ -8,10 +8,10 @@ weight: 3
 This page documents the networking communication between components in the Longhorn system. Using this information, users can write Kubernetes NetworkPolicy
 to control the inbound/outbound traffic to/from Longhorn components. This helps to reduce the damage when a malicious pod breaks into the in-cluster network.
 
-We have provided some NetworkPolicy example yamls at [here](https://github.com/longhorn/longhorn/tree/master/examples/network-policy).
-Or you can enable the setting in the helm chart to install these NetworkPolicy [https://github.com/longhorn/longhorn/blob/master/chart/values.yaml]
+The helm chart will install NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
+The manifests of these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```
Comment (jhkrug) [inline content/docs/1.5.1/references/networking.md:12]:
```suggestion
The helm chart installs NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
Manifests for these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```

File: content/docs/1.5.2/references/networking.md [hunk 3]
```diff
@@ -8,10 +8,10 @@ weight: 3
 This page documents the networking communication between components in the Longhorn system. Using this information, users can write Kubernetes NetworkPolicy
 to control the inbound/outbound traffic to/from Longhorn components. This helps to reduce the damage when a malicious pod breaks into the in-cluster network.
 
-We have provided some NetworkPolicy example yamls at [here](https://github.com/longhorn/longhorn/tree/master/examples/network-policy).
-Or you can enable the setting in the helm chart to install these NetworkPolicy [https://github.com/longhorn/longhorn/blob/master/chart/values.yaml]
+The helm chart will install NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
+The manifests of these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```
Comment (jhkrug) [inline content/docs/1.5.2/references/networking.md:12]:
```suggestion
The helm chart installs NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
Manifests for these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```

File: content/docs/1.5.3/references/networking.md [hunk 4]
```diff
@@ -8,10 +8,10 @@ weight: 3
 This page documents the networking communication between components in the Longhorn system. Using this information, users can write Kubernetes NetworkPolicy
 to control the inbound/outbound traffic to/from Longhorn components. This helps to reduce the damage when a malicious pod breaks into the in-cluster network.
 
-We have provided some NetworkPolicy example yamls at [here](https://github.com/longhorn/longhorn/tree/master/examples/network-policy).
-Or you can enable the setting in the helm chart to install these NetworkPolicy [https://github.com/longhorn/longhorn/blob/master/chart/values.yaml]
+The helm chart will install NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
+The manifests of these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```
Comment (jhkrug) [inline content/docs/1.5.3/references/networking.md:12]:
```suggestion
The helm chart installs NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
Manifests for these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```

File: content/docs/1.5.4/references/networking.md [hunk 5]
```diff
@@ -8,10 +8,10 @@ weight: 3
 This page documents the networking communication between components in the Longhorn system. Using this information, users can write Kubernetes NetworkPolicy
 to control the inbound/outbound traffic to/from Longhorn components. This helps to reduce the damage when a malicious pod breaks into the in-cluster network.
 
-We have provided some NetworkPolicy example yamls at [here](https://github.com/longhorn/longhorn/tree/master/examples/network-policy).
-Or you can enable the setting in the helm chart to install these NetworkPolicy [https://github.com/longhorn/longhorn/blob/master/chart/values.yaml]
+The helm chart will install NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
+The manifests of these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```
Comment (jhkrug) [inline content/docs/1.5.4/references/networking.md:12]:
```suggestion
The helm chart installs NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
Manifests for these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```

File: content/docs/1.5.5/references/networking.md [hunk 6]
```diff
@@ -8,10 +8,10 @@ weight: 3
 This page documents the networking communication between components in the Longhorn system. Using this information, users can write Kubernetes NetworkPolicy
 to control the inbound/outbound traffic to/from Longhorn components. This helps to reduce the damage when a malicious pod breaks into the in-cluster network.
 
-We have provided some NetworkPolicy example yamls at [here](https://github.com/longhorn/longhorn/tree/master/examples/network-policy).
-Or you can enable the setting in the helm chart to install these NetworkPolicy [https://github.com/longhorn/longhorn/blob/master/chart/values.yaml]
+The helm chart will install NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
+The manifests of these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```
Comment (jhkrug) [inline content/docs/1.5.5/references/networking.md:12]:
```suggestion
The helm chart installs NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
Manifests for these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```

File: content/docs/1.5.6/references/networking.md [hunk 7]
```diff
@@ -8,10 +8,10 @@ weight: 3
 This page documents the networking communication between components in the Longhorn system. Using this information, users can write Kubernetes NetworkPolicy
 to control the inbound/outbound traffic to/from Longhorn components. This helps to reduce the damage when a malicious pod breaks into the in-cluster network.
 
-We have provided some NetworkPolicy example yamls at [here](https://github.com/longhorn/longhorn/tree/master/examples/network-policy).
-Or you can enable the setting in the helm chart to install these NetworkPolicy [https://github.com/longhorn/longhorn/blob/master/chart/values.yaml]
+The helm chart will install NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
+The manifests of these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```
Comment (jhkrug) [inline content/docs/1.5.6/references/networking.md:12]:
```suggestion
The helm chart installs NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
Manifests for these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```

File: content/docs/1.6.0/references/networking.md [hunk 8]
```diff
@@ -8,10 +8,10 @@ weight: 3
 This page documents the networking communication between components in the Longhorn system. Using this information, users can write Kubernetes NetworkPolicy
 to control the inbound/outbound traffic to/from Longhorn components. This helps to reduce the damage when a malicious pod breaks into the in-cluster network.
 
-We have provided some NetworkPolicy example yamls at [here](https://github.com/longhorn/longhorn/tree/master/examples/network-policy).
-Or you can enable the setting in the helm chart to install these NetworkPolicy [https://github.com/longhorn/longhorn/blob/master/chart/values.yaml]
+The helm chart will install NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
+The manifests of these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```
Comment (jhkrug) [inline content/docs/1.6.0/references/networking.md:12]:
```suggestion
The helm chart installs NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
Manifests for these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```

File: content/docs/1.6.1/references/networking.md [hunk 9]
```diff
@@ -8,10 +8,10 @@ weight: 3
 This page documents the networking communication between components in the Longhorn system. Using this information, users can write Kubernetes NetworkPolicy
 to control the inbound/outbound traffic to/from Longhorn components. This helps to reduce the damage when a malicious pod breaks into the in-cluster network.
 
-We have provided some NetworkPolicy example yamls at [here](https://github.com/longhorn/longhorn/tree/master/examples/network-policy).
-Or you can enable the setting in the helm chart to install these NetworkPolicy [https://github.com/longhorn/longhorn/blob/master/chart/values.yaml]
+The helm chart will install NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
+The manifests of these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```
Comment (jhkrug) [inline content/docs/1.6.1/references/networking.md:12]:
```suggestion
The helm chart installs NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
Manifests for these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```

File: content/docs/1.6.2/references/networking.md [hunk 10]
```diff
@@ -8,10 +8,10 @@ weight: 3
 This page documents the networking communication between components in the Longhorn system. Using this information, users can write Kubernetes NetworkPolicy
 to control the inbound/outbound traffic to/from Longhorn components. This helps to reduce the damage when a malicious pod breaks into the in-cluster network.
 
-We have provided some NetworkPolicy example yamls at [here](https://github.com/longhorn/longhorn/tree/master/examples/network-policy).
-Or you can enable the setting in the helm chart to install these NetworkPolicy [https://github.com/longhorn/longhorn/blob/master/chart/values.yaml]
+The helm chart will install NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
+The manifests of these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```
Comment (jhkrug) [inline content/docs/1.6.2/references/networking.md:12]:
```suggestion
The helm chart installs NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
Manifests for these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```

File: content/docs/1.6.3/references/networking.md [hunk 11]
```diff
@@ -8,10 +8,10 @@ weight: 3
 This page documents the networking communication between components in the Longhorn system. Using this information, users can write Kubernetes NetworkPolicy
 to control the inbound/outbound traffic to/from Longhorn components. This helps to reduce the damage when a malicious pod breaks into the in-cluster network.
 
-We have provided some NetworkPolicy example yamls at [here](https://github.com/longhorn/longhorn/tree/master/examples/network-policy).
-Or you can enable the setting in the helm chart to install these NetworkPolicy [https://github.com/longhorn/longhorn/blob/master/chart/values.yaml]
+The helm chart will install NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
+The manifests of these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```
Comment (jhkrug) [inline content/docs/1.6.3/references/networking.md:12]:
```suggestion
The helm chart installs NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
Manifests for these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```

File: content/docs/1.7.0/references/networking.md [hunk 12]
```diff
@@ -8,10 +8,10 @@ weight: 3
 This page documents the networking communication between components in the Longhorn system. Using this information, users can write Kubernetes NetworkPolicy
 to control the inbound/outbound traffic to/from Longhorn components. This helps to reduce the damage when a malicious pod breaks into the in-cluster network.
 
-We have provided some NetworkPolicy example yamls at [here](https://github.com/longhorn/longhorn/tree/master/examples/network-policy).
-Or you can enable the setting in the helm chart to install these NetworkPolicy [https://github.com/longhorn/longhorn/blob/master/chart/values.yaml]
+The helm chart will install NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
+The manifests of these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```
Comment (jhkrug) [inline content/docs/1.7.0/references/networking.md:12]:
```suggestion
The helm chart installs NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
Manifests for these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```

**Review by jhkrug — CHANGES_REQUESTED** [review]
A minor suggestion.

**Review by jhkrug — APPROVED** [review]
lgtm

File: content/docs/1.5.4/references/networking.md [hunk 13]
```diff
@@ -8,10 +8,10 @@ weight: 3
 This page documents the networking communication between components in the Longhorn system. Using this information, users can write Kubernetes NetworkPolicy
 to control the inbound/outbound traffic to/from Longhorn components. This helps to reduce the damage when a malicious pod breaks into the in-cluster network.
 
-We have provided some NetworkPolicy example yamls at [here](https://github.com/longhorn/longhorn/tree/master/examples/network-policy).
-Or you can enable the setting in the helm chart to install these NetworkPolicy [https://github.com/longhorn/longhorn/blob/master/chart/values.yaml]
+The helm chart installs NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
```
Comment (innobead) [inline content/docs/1.5.4/references/networking.md:11]: Please update the file path using the corresponding version. Also, update the doc for other versions.

https://github.com/longhorn/longhorn/blob/v{{< current-version >}}/chart/values.yaml

**Review by innobead — CHANGES_REQUESTED** [review]

File: content/docs/1.5.4/references/networking.md (hunk 13 above)
Comment (lahwaacz) [reply content/docs/1.5.4/references/networking.md]: Done.

**Review by innobead — APPROVED** [review]

**Pull request merged**